models/stg_google_play__stats_ratings_app_version.sql

```diff
@@ -0,0 +1,35 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_google_play__stats_ratings_app_version_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_google_play__stats_ratings_app_version_tmp')),
+                staging_columns=get_stats_ratings_app_version_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        date as date_day,
+        app_version_code,
+        package_name,
+        -- an average of an average will only be taken for NULL app versions :-) as they are being grouped together
```

Comment:
I'm skeptical about this calculation. Mathematically, the average of averages is not the true average, unless each average is over a data set of the same size and therefore could make this metric wrongly interpreted. If it is an incorrect calculation, is it worth including?

Reply:
yeah i agree, but i'm unsure how to otherwise aggregate NULL-app version records, since we don't have the number of ratings or anything....